Tests.Unit.Presentation.Web/Services/ItSystemUsageMigrationServiceTest.cs

```diff
@@ -705,6 +765,53 @@ private void ExpectAllowSystemMigrationReturns(bool value)
             _authorizationContext.Setup(x => x.HasPermission(It.IsAny<SystemUsageMigrationPermission>())).Returns(value);
         }
 
+        private void ExpectSystemUsageServiceToRemoveRelationInterface(SystemRelation relation, bool shouldSucceed)
+        {
+            var newRelation = new SystemRelation(relation.FromSystemUsage)
+            {
+                Id = relation.Id,
+                Description = relation.Description,
+                AssociatedContract = relation.AssociatedContract,
+                AssociatedContractId = relation.AssociatedContractId,
+                Reference = relation.Reference,
+                ToSystemUsage = relation.ToSystemUsage,
+                ToSystemUsageId = relation.ToSystemUsageId,
+                UsageFrequency = relation.UsageFrequency,
+                UsageFrequencyId = relation.UsageFrequencyId,
+                RelationInterfaceId = null,
+                RelationInterface = null
+            };
+            if (shouldSucceed)
+            {
+                _itSystemUsageService
```

Comment:
høhø--- hvorfor gentager du dig selv?
refactor de to blokke til at skifte på shouldSucceed ? Result<SystemRelation, OperationError>.Success(newRelation) : Result<SystemRelation, OperationError>.Failure(new OperationError(OperationFailure.UnknownError))